en_US/modules/ocpp_protocol.md

```diff
@@ -0,0 +1,159 @@
+# OCPP Protocol Gateway
+
+## Protocol Introduction
+
+[OCPP (Open Charge Point Protocol)](https://www.openchargealliance.org/)
+is a communication protocol used between Charge Points and a Central System to
+enable the monitoring and control of EV (Electric Vehicle) charging sessions.
+
+Since e4.4.18, EMQX has implemented the `emqx_ocpp` plugin based on the
+[OCCP-J 1.6](https://www.openchargealliance.org/protocols/ocpp-16/) standard, providing the ability
+to connect to Charge Point devices.
+
+::: note
+The current implementation is only provided as a plugin and does not support modules at this time.
+:::
+
+## Quick Start
+
+All configurations of the OCPP plugin are located in the `plugins/emqx_ocpp.conf` file under the
+EMQX etc directory.
+
+Just execute the following command to start the OCPP plugin with default configuration file:
+```bash
+emqx_ctl plugins load emqx_ocpp
+```
+
+### Connect to OCPP Gateway
+
+After the plugin is successfully loaded, you can use [ocpp-go](https://github.com/lorenzodonini/ocpp-go)
+to simulate a Charge Point for connection testing.
+
+Please follow the instructions provided in the ocpp-go documentation to complete the compilation
+first, and then execute the following command to connect to EMQX's OCPP gateway.
+
+Note: `<host>` needs to be modified to the address of your actual EMQX node.
+```bash
+CLIENT_ID=chargePointSim CENTRAL_SYSTEM_URL=ws://<host>:33033/ocpp go run example/1.6/cp/*.go
+```
+
+## ClientInfo and connection
+
+The OCPP gateway only accept the clients by Websocket, as defined in the OCPP-J 1.6 protocol.
+
+Once the connection is established successfully, EMQX treats it as a standard client. This means
+that you can manage this client through Client ID on Dashboard/HTTP-API/CLI.
+
+The mapping of client information is as follows:
+- Client ID: The unique identifier of the Charge Point.
+- Username: Obtained from HTTP Basic authentication provided by the Charge Point during connection.
+- Password: Also obtained from HTTP Basic authentication.
+
+
+### Authentication
+
+As mentioned in the OCPP-J 1.6 specification, Charge Points can use HTTP Basic for authentication.
```

Comment:
I'd suggest to rewrite this whole paragraph as "EMQX OCPP gateway extracts the user name and password from the HTTP Basic authentication used by Charge Points (as specified in OCPP-J 1.6), and then uses EMQX's authentication system to manage the client and determine access permission."